.travis.yml

```diff
@@ -0,0 +1,49 @@
+language: julia
+
+os:
+  - linux
+  - osx
+  - freebsd
+  - windows
+
+julia:
+  - 1.0
+  - 1.4
```

Comment:
I find using `1` is better because it will automatically roll to `1.5` when it is released avoiding you having to go through and PR the change every release!